source/schemas/index.rst

```diff
@@ -17,3 +17,13 @@ must support the "core" module, but the other modules are optional.
    unit.rst
    transform.rst
    wcs.rst
+
+The following graph shows the dependencies between modules:
+
+.. digraph:: modules
+
+   "fits" -> "core"
+   "unit" -> "core"
+   "transform" -> "core"
+   "wcs" -> "transform"
+   "wcs" -> "unit"
```

Comment:
That's not a bad idea.  They are explicitly documented in the .rst files that pull in the schema documentation, but having it the YAML would let tools programmatically get at the info.